src/DatabaseSchemaManager.php

```diff
@@ -0,0 +1,251 @@
+    {
+        if (! $this->validateInputs($connectionName, $tableName, $columnName)) {
+            return true;
+        }
+
+        $column = current(array_filter($this->schema[$connectionName][$tableName], function ($column) use ($columnName) {
+            return $column['name'] === $columnName;
+        }));
+
+        return $column['is_nullable'];
+    }
+
+    /**
+     * Check if the provided data is valid.
+     *
+     * @param  string  $connetionName
+     * @param  string  $tableName
+     * @param  string  $columnName
+     * @return bool
+     */
+    private function validateInputs($connectionName, $tableName, $columnName)
+    {
+        if ($this->ensureSchemaExistence($connectionName) &&
+            $this->ensureTableExistence($connectionName, $tableName) &&
+            $this->ensureColumnExistence($connectionName, $tableName, $columnName)) {
+            return true;
+        }
+
+        return false;
+    }
+
+    /**
+     * Check if the column has default value set on database.
+     *
+     * @param  string  $connetionName
+     * @param  string  $tableName
+     * @param  string  $columnName
+     * @return bool
+     */
+    public function columnHasDefault($connectionName, $tableName, $columnName)
+    {
+        if (! $this->validateInputs($connectionName, $tableName, $columnName)) {
+            return false;
+        }
+
+        $column = current(array_filter($this->schema[$connectionName][$tableName], function ($column) use ($columnName) {
+            return$column['name'] === $columnName;
+        }));
+
+        return $column['default'] !== null ? true : false;
+    }
+
+    /**
+     * Get the default value for a column on database.
+     *
+     * @param  string  $connetionName
+     * @param  string  $tableName
+     * @param  string  $columnName
+     * @return bool
+     */
+    public function getColumnDefault($connectionName, $tableName, $columnName)
+    {
+        if ($this->columnHasDefault($connectionName, $tableName, $columnName)) {
+            $column = current(array_filter($this->schema[$connectionName][$tableName], function ($column) use ($columnName) {
+                return $column['name'] === $columnName;
+            }));
+
+            return $column['default'];
+        }
+
+        return null;
+    }
+
+    /**
+     * Check if the table exists in database.
+     *
+     * @param  string  $connetionName
+     * @param  string  $tableName
+     * @return bool
+     */
+    public function hasTable($connectionName, $tableName)
+    {
+        if (! $this->ensureSchemaExistence($connectionName)) {
+            return false;
+        }
+
+        return isset($this->schema[$connectionName][$tableName]);
+    }
+
+    /**
+     * Make sure table exists or throw an exception.
+     *
+     * @param  string  $connetionName
+     * @param  string  $tableName
+     * @return bool
+     */
+    private function ensureTableExistence($connectionName, $tableName)
```

Comment:
```suggestion
    private function tableExists($tableName, $connectionName)
```

Wouldn't this be easier to read?